travis/update-release.py

```diff
@@ -179,66 +205,38 @@ def check_rollback_artifacts(r_stream,tag):
             # Change the working directory temporarily
             os.chdir("/tmp/" + GIT_LOCAL_DIR)
 
-            # Command 1: git stash
-            subprocess.run(["git", "stash"], universal_newlines=True)
+            # Run the git workflow
+            run_git_commands(quaySha)
 
-            # Command 2: git log --grep=xyz --format=%H
-            git_log_command = ["git", "log", "--grep=" + quaySha, "--format=%H"]
-            commit_hash = subprocess.check_output(git_log_command, universal_newlines=True).strip()
+            # Return to the original working directory
+            os.chdir(current_directory)
 
-            # Command 3: git checkout abcd
-            subprocess.run(["git", "checkout", commit_hash], universal_newlines=True)
+            image_update = create_release_image_data(image, dockerSha,quaySha,tag)
 
-            # Command 4: copy some files to a destination (e.g., using shutil or any other method)
-            copyfile("docs/release_artifacts/" + RELEASE_TAG + "/z/" + image["name"],
-                            "/tmp/z/"+image["name"])
+            c_images.append(image_update)
+
+        return c_images, True
+    elif new_sync:
```

Comment:
good point. The intention was that once we have all up to date images to z releases , anything after this if that's a pull and push of a z image for release tag , in that case this code handles the case. Now when we doing the rollback and pushing new sha at the same time and then running release, yes it will go to latest z releases and avoids the rollback case. Let me check here